iorgen/utils.py

```diff
@@ -30,6 +30,11 @@ def int_to_iterator_name(value: int, times: int = 1) -> str:
     return int_to_iterator_name(value - 18, times + 1)
 
 
+def str_int(value: int) -> str:
+    """Return integer mathjax representation"""
+    return f"{value:,}".replace(',', r'\,')
```

Comment:
So we’ll have no comma in the TeX rendering, only spaces, right?

Reply:
Yes, I use the f-string to make Python format the number (which will result in a number in English format, with commas) and then I replace every comma with `\,` which is the Mathjax notation for thin space